src/arch/riscv/RiscvISA.py

```diff
@@ -52,9 +55,19 @@ class RiscvISA(BaseISA):
     cxx_class = "gem5::RiscvISA::ISA"
     cxx_header = "arch/riscv/isa.hh"
 
+    system = Param.System(Parent.any, "System this ISA object belongs to")
+
     check_alignment = Param.Bool(
         True, "whether to check memory access alignment"
     )
+
     riscv_type = Param.RiscvType("RV64", "RV32 or RV64")
 
     enable_rvv = Param.Bool(True, "Enable vector extension")
+
+    riscv_vlen_se = Param.RiscvVectorLength(
+        256, "RISC-V maximum vector length"
```

Comment:
Should be 4096, SiFive already implemented x280 which has VLEN=512b, and Andes also has a core with VLEN=1024b

Ref: https://github.com/riscv/riscv-v-spec/issues/204

Reply:
Could you expand on why you think we should make this change? This is a default for a parameter, not something that is set in stone.

Maybe the description could be updated to say something like "the hardware length of registers, VLEN in the spec"

Reply:
@jerryzj I want to clarify that this parameter supposedly define an architecture's VLEN, not the maximum VLMAX. The VLEN parameter can be changed by user.

Reply:
@hnpl got it, I misunderstood the meaning of the variable here, thanks for the clarification.
However, I would also suggest that the default VLEN could be set to 128b as it is the default length of V extension.